Apsitvarkom.DataAccess/IEntityDTORepository.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Apsitvarkom.DataAccess;
+
+/// <summary>
+/// Interface for <see cref="T" /> class data handling.
+/// </summary>
+public interface IEntityDTORepository<T> where T: class
```

Comment:
Any specific reason why `Entity` here in the name? I would suggest not adding one more word to our code just to not confuse people. Usually more words for the same thing = more confusion.

What about `IModelDTORepository`, `IDTORepository` or just `IRepository` ?